src/main/build/atomic.h

```diff
@@ -142,7 +142,7 @@ static inline uint8_t __basepriSetRetVal(uint8_t prio)
 // On gcc 5 and higher, this protects only memory passed as parameter (any type can be used)
 // this macro can be used only ONCE PER LINE, but multiple uses per block are fine
 
-#if (__GNUC__ > 6)
+#if (__GNUC__ > 7)
 # warning "Please verify that ATOMIC_BARRIER works as intended"
```

Comment:
Have you verified that?

Reply:
i didnt checked if the assembly is exactly the same. 
if it would not works as intended betaflight would be broken (since this is used at different important places like timers) and since this isnt the case and betaflight compiled with gcc7 works fine it works as intended.

Reply:
I think the warning was added by @ledvinap, so probably better ask him. Just because your tests have not seen the firmware fail miserably does not mean that all is ok, otherwise we would not be having this warning.

Reply:
inital @hydra created that warning. if any one want to review this in depth i would appricate it ;)
how the time flys by ;) https://github.com/betaflight/betaflight/pull/1917